sceptre/resolvers/file.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
+
+import os
+import json
+import yaml
+import requests
```

Comment:
is requests in the python 3.6 standard library?

Reply:
No it's a dependency.